semantic-python/src/Prelude.score

```diff
@@ -1,8 +1,22 @@
 {
-  type <- \name -> \bases -> \dict ->
-    #record { __name: name, __bases: bases, __dict: dict };
+  type <- \name -> \super -> \slots ->
+    #record { __name: name, __super: super, __slots: slots };
```

Comment:
_The Full Monty_ defines classes as a triple containing a class name, its superclasses (one or more), and a slots field for arbitrary assignment.